azure-pipelines.yml

```diff
@@ -14,7 +14,7 @@ stages:
   jobs:
     - job: Skip
       pool:
-        vmImage: 'ubuntu-18.04'
+        vmImage: 'ubuntu-latest'
```

Comment:
Question - what does this job do?

Reply:
Just decides if all other steps should be skipped based on `[skip azp]` or `[azp skip]` being in the commit message

Reply:
Thanks, sorry my mis-understanding was more why would we need to specify/change the vmImage since this is just doing the skip?

Reply:
Eventually we'd need to once they retire 18.04. by using latest, we hopefully won't need to update it again